.github/workflows/format.yml

```diff
@@ -49,11 +49,15 @@ jobs:
     name: kotlin_lint
     runs-on: ubuntu-18.04
     timeout-minutes: 45
-    container:
-      image: openjdk:8-jdk
     steps:
       - uses: actions/checkout@v1
         with:
           submodules: true
-      - name: 'Run Kotlin lint'
-        run: ./ci/kotlin_lint.sh
+      - name: 'Install dependencies'
+        run: ./ci/linux_ci_setup.sh
+      - name: 'Run Kotlin Lint'
```

Comment:
```suggestion
      - name: 'Run Detekt (Kotlin Lint)'
```

Reply:
The more info about the actual tool we have in the names of things the easier it is to work with for external contributors, ie when they see a Detekt failure but don't know that it's Detekt it might be confusing

Reply:
Let’s hear @buildbreaker. I was under an impression that the Lint tool was intentionally abstracted — even the config file is called `kotlinlint.yml`.

Reply:
I think it'll be nice to abstract it away. We could do something like `Run Kotlin Lint (Detekt)`

Reply:
`Run Kotlin Lint (Detekt)` lgtm, but I'm not blocking on that, use your judgement :)

Reply:
Replaced with `Run Kotlin Lint (Detekt)` and `Run Swift Lint (SwiftLint)` for consistency.